ExampleMod/Content/Projectiles/Minions/ExampleHoveringMinion.cs

```diff
@@ -0,0 +1,88 @@
+using ExampleMod.Dusts;
+using Microsoft.Xna.Framework;
+using Terraria;
+using Terraria.ID;
+using static Terraria.ModLoader.ModContent;
```

Comment:
It has been decided that 'using static' definitions shouldn't be in ExampleMod, to avoid confusion.